web/src/engine/configuration/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#
+
+# set -x
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+WORKING_DIRECTORY=`pwd`
+
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
+
+# Imports common Web build-script definitions & functions
+SUBPROJECT_NAME=engine/configuration
+. ../../../.common.sh
```

Comment:
```suggestion
. "$KEYMAN_ROOT/web/common.inc.sh"
```

A little easier to read where it is coming from